spec/models/goal_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Goal, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Consider adding a test for the goal-belongs_to-user association. It'll be like the work you did in bloccit here: https://github.com/jeffminnear/bloccit/blob/93f3b027d2a61cd88006905574a6cafcfb51824f/spec/models/comment_spec.rb#L12